packages/perseus/src/widgets/__tests__/numeric-input_test.jsx

```diff
@@ -50,68 +53,50 @@ const renderQuestion = (question, apiOptions: APIOptions) => {
     return {container, renderer};
 };
 
-const options: $ReadOnlyArray<APIOptions> = [
-    {satStyling: true},
-    {satStyling: false},
-];
-
-describe.each(options)(
-    "numeric-input widget (options: %j)",
-    (apiOptions: APIOptions) => {
-        describe("question", () => {
-            const [question, correct, incorrect] = question1AndAnswer;
-
-            beforeEach(() => {
-                jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
-                    testDependencies,
-                );
-            });
-
-            it("Should accept the right answer", () => {
-                // Arrange
-                const {renderer} = renderQuestion(question, apiOptions);
-
-                // Act
-                userEvent.paste(
-                    screen.getByRole("textbox", {hidden: true}),
-                    correct,
-                );
-
-                // Assert
-                expect(renderer).toHaveBeenAnsweredCorrectly();
-            });
-
-            it("Should render predictably", () => {
-                // Arrange
-                const {container} = renderQuestion(question, apiOptions);
-                expect(container).toMatchSnapshot("first render");
-
-                // Act
-                userEvent.paste(
-                    screen.getByRole("textbox", {hidden: true}),
-                    correct,
-                );
-
-                // Assert
-                expect(container).toMatchSnapshot("after interaction");
-            });
-
-            it("should reject an incorrect answer", () => {
-                // Arrange
-                const {renderer} = renderQuestion(question, apiOptions);
-
-                // Act
-                userEvent.paste(
-                    screen.getByRole("textbox", {hidden: true}),
-                    incorrect,
-                );
-
-                // Assert
-                expect(renderer).toHaveBeenAnsweredIncorrectly();
-            });
-        });
-    },
-);
+describe("numeric-input widget", () => {
+    const [question, correct, incorrect] = question1AndAnswer;
+    const apiOptions = Object.freeze({});
```

Comment:
Since `renderQuestion` has a default for `apiOptions` and in this section `apiOptions` is just the same value as that default, I think it would make these tests clearer if we removed `apiOptions` entirely from this section. Future devs will get to this line of code and think we're using `apiOptions` for something related to the tests, but it doesn't look like we are.

Reply:
Good call! Thanks.